plugin/fanout/README.md

```diff
@@ -0,0 +1,108 @@
+# fanout
+
+## Name
+
+*fanout* - parallel proxying DNS messages to upstream resolvers.
+
+## Description
+
+Each incoming DNS query that hits the CoreDNS fanout plugin will be replicated in parallel to each listed IP (i.e. the DNS servers). The first non-negative response from any of the queried DNS Servers will be forwarded as a response to the application's DNS request.
+
+## Syntax
+
+* `tls` **CERT** **KEY** **CA** define the TLS properties for TLS connection. From 0 to 3 arguments can be
+  provided with the meaning as described below
+  * `tls` - no client authentication is used, and the system CAs are used to verify the server certificate
+  * `tls` **CA** - no client authentication is used, and the file CA is used to verify the server certificate
+  * `tls` **CERT** **KEY** - client authentication is used with the specified cert/key pair.
+    The server certificate is verified with the system CAs
+  * `tls` **CERT** **KEY**  **CA** - client authentication is used with the specified cert/key pair.
+    The server certificate is verified using the specified CA file
+* `tls_servername` **NAME** allows you to set a server name in the TLS configuration; for instance 9.9.9.9
+  needs this to be set to `dns.quad9.net`. Multiple upstreams are still allowed in this scenario,
+  but they have to use the same `tls_servername`. E.g. mixing 9.9.9.9 (QuadDNS) with 1.1.1.1
+  (Cloudflare) will not work.
+
+* `fail-count` is the number of subsequent failed health checks that are needed before client stops trying to connect.
+* `worker-count` is the number of parallel queries per request.
+* `policy` is specifies the policy to use for selecting response from proxies.
+  * `any` is a policy that returns any first response from success connected proxies.
+  * `first-positive` is a policy that returns any first non-negative response from proxies.
```

Comment:
should mention that this is the default option.

Reply:
thanks! Fixed